modules/eco-node-health-check-operator-creating-node-health-check.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-health-check-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-node-health-check-operator-creating-node-health-check_{context}"]
+= Creating a node health check
+Using the web console, you can create a node health check to identify unhealthy nodes and specify the remediation type and strategy to fix them.
+
+.Procedure
+
+. From the *Administrator* perspective of the {product-title} web console, click *Compute* -> *NodeHealthChecks* -> *CreateNodeHealthCheck*.
+. Specify whether to configure the node health check using the *Form view*  or the *YAML view*.
+. Enter a *Name* for the node health check. The name must consist of lower case, alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character.
+. Specify the *Remediator* type, and *Self node remediation* or *Other*. The Self node remediation options are part of the the Self Node Remediation operator that is installed with the Node Health Check operator. Selecting *Other* requires an *API version*, *Kind*, *Name*, and *Namespace* to be entered, which then points to the remediation template resource of a remediator.
```

Comment:
Just one small nit - it should be "The self node remediation option is..." singular and not "options are" since there is no strategy selection